api/src/main/java/org/openmrs/module/fhirExtension/translators/impl/ObsBasedDiagnosticReportTranslatorImpl.java

```diff
@@ -41,7 +41,8 @@ public class ObsBasedDiagnosticReportTranslatorImpl implements ObsBasedDiagnosti
 	@Autowired
 	private ProviderService providerService;
 	
-	static final String PRACTITIONER = "Practitioner";
+	@Autowired
+	private PractitionerReferenceTranslator<Provider> practitionerReferenceTranslator;
```

Comment:
practitionerReferenceTranslator also need to be called from toOpenMrsType function.